Formula/u/uv.rb

```diff
@@ -23,6 +23,11 @@ class Uv < Formula
 
   uses_from_macos "python" => :test
 
+  on_linux do
+    # We only ship bzip2.pc on Linux which bzip2-sys needs to find library
+    depends_on "bzip2"
+  end
```

Comment:
this would be better.
```suggestion
  uses_from_macos "bzip2"
```

Reply:
It doesn't use macOS `bzip2`.

It uses the bundled copy.

Reply:
i see 👍